governance/contributor-ladder.md

```diff
@@ -0,0 +1,112 @@
+While it is essential to provide support to current members and leaders, it is equally important to focus on assisting and orienting newcomers.
+Modeling empathy and offering support to the community as a whole should be at the forefront of your actions.
+It is crucial to role model the values of the community as outlined in the [code of conduct](http://cncf.io/conduct/).
+
+There are various ways you can contribute to supporting and empowering new contributors.
+You can actively participate by offering mentorship and guidance when someone requests assistance. This could involve helping them navigate the community, providing technical guidance, or answering their questions.
+Additionally, if a member is unable to fulfill their responsibilities, you can offer to fill in their role temporarily or assist in any necessary tasks.
+Co-hosting meetups and events is another way to provide support. By sharing the responsibility, you help create an inclusive environment where everyone feels supported and encouraged.
+
+Building a brand of being someone who consistently helps is highly valuable.
+Actions to help this include engaging in discussions, providing feedback on proposals, reviewing code contributions, and offering assistance in any way that aligns with your skills and expertise.
+By actively participating and contributing, you demonstrate your commitment to the community and showcase the values that the Environmental Sustainability TAG holds dear.
+
+* Helping out and mentor new contributors in their open source journey.
+* Helping out community members and leaders if they need support.
+* Demonstrating empathy and embody the community values.
+
+## Leading a Working Group
+
+Since leadership roles naturally change over time, it is worth your while to take a leadership position in an established working group.
+Within the TAG for Environmental Sustainability, dedicated working groups carry out several ongoing initiatives.
+However, if the current working groups do not provide space for the topic you are interested in, you could start a new working group.
+A working group must have community support and requires that you find sufficient backing before a working group is formed.
+To start this process, create a GitHub issue describing the initial charter in which you outline the motivation, scope, goals, and deliverables of the proposed working group.
+This working group charter document should form the basis for discussions with the TAG before initiating an official vote by the TAG co-chairs, which also requires approval by the CNCF TOC.
+It is important to present the working group proposal during TAG meetings and be vocal about the discussions, actively solicit feedback and input, and gain support from other members.
+
+* Taking on more responsibility and ownership within TAG.
+* Leading a community group within TAG.
+* Conflict resolution.
```

Comment:
These three bullets kinda are sitting there without a lot of context, now that I'm re-reading.  Do we want to say "pre-requisites for leaders"?

```suggestion
* Good at negotiating with differing opinions.
```

I try not to suggest there will be conflicts, but it is good to have something similar to that here.